discovery/spec.md

```diff
@@ -390,6 +390,9 @@ The following sections define the attributes that appear in a Service entity.
 
 The following sections define the attributes that appear in EventType definitions of a Service object.
 
+*Note:* while attributes elsewhere are camelCased, the attributes within `EventType` are all lower-case,
+as they correspond directly to CloudEvent attributes (with the exception of `extensions`).
+
```

Comment:
Makes sense.

Reply:
Been thinking about this since our last call... while it's less pretty, I'm very worried about the pain our users will experience in our perceived inconsistency here. While having everything as lowercase isn't ideal, at least it doesn't require people to think about which set of attributes are camel-cased and which are not.

Reply:
Right. Let's take a straw poll at the meeting tomorrow.

A thoroughly-considered decision to stay as we are is absolutely fine by me - I just didn't want to get there accidentally.